cyclonedx-bom/src/models/bom.rs

```diff
@@ -85,7 +92,49 @@ impl Bom {
         let config = EmitterConfig::default().perform_indent(true);
         let mut event_writer = EventWriter::new_with_config(writer, config);
 
-        let bom: crate::specs::v1_3::bom::Bom = self.into();
+        let bom: crate::specs::v1_3::bom::Bom = self
+            .try_into()
+            .expect("todo: error handling in output_as_json_v1_3");
+        bom.write_xml_element(&mut event_writer)
+    }
+
+    /// Parse the input as a JSON document conforming to [version 1.4 of the specification](https://cyclonedx.org/docs/1.4/json/)
+    pub fn parse_from_json_v1_4<R: std::io::Read>(
```

Comment:
This is not necessarily an issue with the code, could be my ignorance of the codebase and/or the format, but: is there any reason to keep the 1.3 methods around now that we have 1.4 ones? Isn't 1.4 a strict superset with no new mandatory fields, so that 1.3 data could be parsed by methods expecting 1.4?